src/store/modules/pickups.js

```diff
@@ -239,6 +239,8 @@ export function isWithinOneWeek (pickup) {
   return pickup.date < new Date(+new Date() + 6096e5)
 }
 
-export function sortByDate (a, b) {
-  return a.date > b.date
```

Comment:
Would it be sufficient to replace the line with this:
```
return a.date - b.date;
```

It seems that `a.date > b.date` behaved wrong because it returned true or false instead of integers.